assets/app/views/browse/build-config.html

```diff
@@ -300,6 +297,19 @@
                                 <dt>Output to:</dt>
                                 <dd>{{buildConfig.spec.output.to | imageObjectRef : buildConfig.metadata.namespace}}</dd>
                               </div>
+                              <div class="run-policy">
+                                <dt>Run Policy:</dt>
+                                <dd>
+                                  {{buildConfig.spec.runPolicy | sentenceCase}}
+                                  <span class="help action-inline">
+                                    <a href ng-switch="buildConfig.spec.runPolicy">
```

Comment:
if they add a new runPolicy that we don't know about we could end up with an empty `<a>` here, I would add an ng-switch-default that says `Builds triggered from this Build Configuration will run using the {{buildConfig.spec.runPolicy | sentenceCase}} policy.` its not the most helpful help in that case but it would at least not be broken :)